packages/suite-desktop/scripts/dev.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable */
```

Comment:
i would rater disable one rule (require) than the whole eslint

Reply:
3d8fd3b